### DISCUSSION TITLE
Harder raid difficulty is badly needed.

### DISCUSSION THREAD

**Discussion by IskrenPlashkov** [chromiecraft/chromiecraft#5925] [General]
Last night it became brutally apparent to everybody that the T4 raids are way too easy. They simply provide no challenge whatsoever. With its current state people will get bored of them very soon. It is fine to let newer and less experienced people have the ability to clear content, but everyone else need to at least feel some pushback on their quest to the next set of epics.

With that premise in mind I want to raise the question again of mythic difficulty for raids. Some real tuning is needed if we want to make raids great again. Vanilla raids were working great there and personally I'd like to see a return to similar difficulty - AQ40 is a prime example. In my opinion the wrong lessons were learnt from that raid. It was not the overwhelming difficulty that was the problem, more like 2 buggy bosses and and overtuned patch of trash. 

To summarize have AQ40-esque difficulty as optional mythic challenge for people who want to try it.

**IF** this is implemented however we do need some rewards for the extra efforts required to down mythic content. It will be fun once or twice but it is human nature to revert back to the path of least resistance. Rewards can be in the form of better stats on items, more items dropping from each boss or some other implementation. At the end of the day though some mythic specific reward system is **required** in order to keep the mythic system alive.

Let me know what you guys think :)

**EDIT:** Wow, thanks guys for all the replies and feedback! Reading everything I have reconsidered my position a bit:

While there is some good argument to be made for having 1 middle or the road raid difficulty, similar to how MC and BWL were, I get the idea server staff has of introducing multiple raid difficulties. After all this is the road Blizzard took themselves back in the day. So my stance on the absolute necessity of introducing mythic remains unchanged. 

As stated before harder difficulties need better reward else nobody will bother with them past the 1st attempt. Potential rewards can vary:

- **Increased stats** - Extra gem slot or its stats equivalent seems to be enough and from previous announcements server staff was heading in that way as well. After contemplating this a lot I realized this was not as great as I initially thought, as it would disturb the balance between regular epics and tier pieces - indirectly nerfing the latter. Moreover this has really sparked some outrage in big parts of the community. Some say 51.7% of the server is against item modifications, I call it half and half. When so few have participated in the survey having little over a percent above means nothing. That being said, this is not a hill I am willing to die on. Threatening to quit the server for not getting your way is childish behavior that should not be addressed, especially when we are trying to find the best course of action for a feature that will stick with the server for the rest of its lifespan. 
- **Bigger loot table** - 2 more epics and 1 extra tier piece seem to be good middle of the road solution that still provides good incentive to do mythics while satisfying the no-changes purists.
- **Special currency** - Similar to the original idea or mythic dungeons barring the increased stats. Imho the initial rewards were barely enough to make me want to do mythics regularly, Removing stats from items pushes this over the line. Moreover going for bigger loot table would make this redundant, as players would get much better changes in getting what they want.

To summarize my suggestions I still think mythic raid difficulty is badly needed, with more items dropping from each boss being the best way going forward. 
Should normals be tuned up a bit or not I'd let the more casual guilds to decide, with my 2 cents on the topic being that the current roflstomp state of affairs' is fun for nobody.

**Comment by UndoUreche** [comment]
I prefer the idea of more loot rather than better loot as well. It doesn't force people to do only the hardest content if they want to be at the same level as everyone else, but it also incentivizes guilds to try mythic to gear more of their members faster.

+1 from me

**Comment by shaszun** [comment]
How is this topic still open after 3h, mine was closed 10 minutes after creation. After sleeping this over and thinking a bit on a fresh head. I'm starting to think they did it on purpose. This way Mythics will be more openly taken and won't have the initial backlash like the dungeons did. Still we will have to choose between a 30 mins raids and probably 2-3h raids with the same rewards. Like every miracle the hype from mythics will be dead in 1 month and no one will do them. People will only raid log and guilds will start falling a part.

**Reply by UndoUreche** [reply]
No, I disagree. 
Firstly even if it takes 2 hours vs 30 minutes, the fact remains that the extra time investment would yield better loot, or preferably imo more loot. This should be enough incentive to make people choose them over normals. 

Secondly, not everyone is like me and plays only one character. Most people, in fact, have multiple alts that they'd likely want gear on. In my experience few of us manage to get bis on one character during a raid bracket, let along multiple of them. I would dare to say that it's very unlikely people will abandon them due to having completed their bis sets. 

Thirdly, raid logging is already an issue, and this measure does not aim to fix it. You can't fix raid logging by making changes to the raids, you have to add more content beside raids if you want that.

**Reply by georgeclamnis** [reply]
What you're saying is true. Personally, as a World of Warcraft player for several years on private servers, I've noticed that servers that don't listen to their community all end up as you mentioned before. I joined this server about 6-7 months ago, and when I saw that there was a polling channel and that the staff was attentive to the community, I quickly understood that this server had potential. Now, I see the opposite: many people are complaining about mythic raids (and they are right). They want to have an experience consistent with Blizzard's, but the staff's response gives me the impression that they will still implement the mythic raids, even if their community disagrees (being stubborn). It's a shame they're doing that...

**Reply by shaszun** [reply]
Even if you have multiple alts for which you want to gear. A lot of guilds are running multiple raids. I think Solaris in classic had like 5 different raid groups per reset. We had 2-3 and at some point, we were raiding almost daily in different dungeon. Gearing alts it's not an issue as long you want to do it. You can even join multiple guilds with your chars, who is going to stop you? With current drop rates, in less than 2 months 25 people can get mostly everything they need, of course you will have exceptions with bad luck or items which all the DPS wants it.

**Reply by IskrenPlashkov** [reply]
George, you clearly have not played Gruul and Mag here. This is nowhere near Blizzlike experience. Never did gear check bosses die within 40 seconds on progression night, killed by people in pre bis. T4 content was simply not scaled for 3.3.5. itemization and more importantly talents. I am arguing for simply have similar experience to what it once was - challenging. I also presented an alternative - if staff wants to keep default raids for more casual guilds simply implement harder difficulties but also present proper rewards so people have reliable and long term incentive of ding them.

**Reply by 55Honey** [reply]
> ... many people are complaining about mythic raids 

Spoiler: the majority likes the idea to run mythics.

> ...even if their community disagrees (being stubborn). It's a shame they're doing that...

See comment above. It's factually wrong. We're actively following the desire of the majority.

**Reply by Dudududy** [reply]
Genuine question because I don't remember: When asked about the interest in mythics in the survey, was it disclosed that it would be with custom abilities?

Personally, I answered that I would try mythic. However, if I could have elaborated the answer, I would have said I would like to probably just try it _once_ for the fun of it, and then continue with Blizzlike content.

**Reply by elthehablo** [reply]
Another spoiler alert: if guilds had 1 difficulty with tuning to cater to high-end raiding guilds people would be madge as well

**Comment by v-mstrs** [comment]
Just have one raid difficulty where it's not piss easy but not impossible to do either. Like classic raids were... No extra dumb fluff.  You don't even have to do the mechanics currently. CC team feels out of touch. Do they even play the same game I wonder.

**Reply by elthehablo** [reply]
Some of them do play, and they do watch realm first fights as in GM mode. But yes, many of the devs are not super active in the high-end raiding guilds

**Comment by elthehablo** [comment]
Mythic difficulty for raids needs to be implemented imo. If that happens, then I'm fine with the normal difficulty staying the way it is now personally. But I hard agree that it MUST have some incentive. I would choose for the same system that was thought up, but instead of being able to buy upgraded loot, you buy regular raid loot. This way we make the people that don't want to see custom gear happy as well. Then the system works as a sort of bad luck protection. In summary, I agree with the post.

**Reply by balleny** [reply]
practically looking into the same direction: either have people receive some currency for turn in of that respective boss loots (IMO i would link it boss by boss) or simply increase the amount of loot dropping from the boss in mythic.

last part would obviously also have a larger effect it the loot tables where how they are supposed to in tbc (fewer items, esp. tokens, dropping)

**Reply by shaszun** [reply]
I think current items count drops from T4 raids is way higher than it's supposed to be. Might be done on purpose from staff, might be bugged. 

We got 3 tokes and 1 random item at King. I think it's supposed to be 1 or 2 tokens + 1 random item.
On gruul we got 3 tokes and 2 weapons. Again normally It should be 1 or 2 tokens + 2 random items.
On Mag, I don't remember exactly, but I think it was again like Gruul drop count.

**Reply by balleny** [reply]
it is indeed a lot higher than supposed to (if aligned with progression)
https://github.com/chromiecraft/chromiecraft/issues/5930

and unfortunately, also a easily missed opportunity to make an incentive for mythic version (with more loot dropping)
https://github.com/chromiecraft/chromiecraft/discussions/5925#discussioncomment-6447762

**Reply by shaszun** [reply]
Current drop count would be nice for mythics for sure. Like +1/2 tokens and +1 random item. But this would be valid only if they nerf the normal difficulty drop count to the blizzlike state.

**Comment by Hogofogo1** [comment]
Heroics are there to gear up for raids and gather badges. Once you have that in about 20 runs you don't want to spend there another minute. There is no point in forcing the hamsterwheel on people with Mythic HCs. We will get plenty of that in Wrath HCs with their Badge system.

Now for raids there are 2 possibilities:
1. Make them normal/hard from the get go and abandon Mythic Difficulty (Blizz way)
2. Make the raids normal and Mythic raids hard with bigger loot table etc. (Custom way)
Both of these are better than super easy versions which isn't Blizz way.

And Blizz way is what we are after aren't we?

**Comment by Kirishiki** [comment]
as we play on wotlk talents the raids should be buffed up to match by defaul. now we have to wait for some mythic mode to feel some challenge and be already max geared at that point by doing the raids on very easy difficulty. sorta ruins the experience.

**Comment by Peppersprayz** [comment]
I can understand easy version of the raids for casual/PUG.
I can understand mythic version of the raids with custom mechanics for tryharders. 
I can understand a AQ40-like tuning of the raids for people that want challenging but "blizzlike" raids.
What i can't understand is why some people seems to think that you can't please everyone whereas it would actually be possible to just have those 3 raid versions instead of just the 2 firsts.

**Comment by TotallyNotMyAccount** [comment]
Frankly put, I couldn't disagree more with some of the opinions I'm reading in here. If you have two difficulties, then the higher difficulty needs to drop better gear. Everything else is not blizzlike. Heroics drop better gear than normal dungeons. Wrath heroic raids drop better loot than normal raids. Arena gives you better loot the higher rated you are and better loot than BGs.
The incentive for doing harder content is better loot. Chromie at the moment feels like a bucket of crabs where the casuals cannot tolerate dedicated players getting better gear than them. If CC staff indulges them they will kill serious raiding guilds and lose a ton of players.

**Reply by shaszun** [reply]
I think you mean only 1 guild and probably only 1 of their raiding groups. I won't mention names, but I think everyone can guess which one is it. This is not an attack or anything, just stating it's not a big chunk of players. I suggest to go outside of your bubble and start thinking about what other people said in this topic.

We are in TBC content right now, how can you compare wrath raid difficulties set by Blizzard themselves with TBC raids? Arena don't give you better gear with higher ratings, it unlocks just another piece of the set and lastly weapons.

Also another thing in wrath between tiers you have big stat difference, which enables you to add the so called heroic raid gear between tiers. In TBC you have 5 stat difference, so you want to get T4 heroic tier piece which is 2 stats better than standard T4? Two months after that to take the T5 which will give you another few stats?

**Reply by TotallyNotMyAccount** [reply]
How can I compare wrath with it's multiple difficulties to tbc content? That's easy, it's clearly what CC staff is emulating with their increased difficulty raids. Blizzard also demonstrated how to do that right: Make it drop better loot. Also, we are playing on a progressive **Wrath** server, so wrath concepts appearing shouldn't be a shock to anyone, since we have been playing the tbc (and vanilla) content with wrath abilities, talents, professions and item rebalances already.

I'm not even lobbying for the crazy item budget inflation blizz started in wrath, what CC had on the ptr with their additional uncommon gem per item was actually a really good spot to be in.

Finally I want to say that I just love how you instantly tried to tarnish me for being in a guild you seem to dislike, while being really thin on anything resembling arguments in general. But please nitpick my shorthand reference to how arena gear works more as if that invalidates the general thrust of my argument.

**Reply by shaszun** [reply]
If you felt my previous comment as an attack, this one might be felt as an attempt to kill you...

Can you and your guildies think for long term about what is good for this server? Because already people are thinking they will F up wrath in the future with all this additional custom things in dungeons and raids. A lot of people stated already if no changes to the normal difficulty is done and mythic includes custom mechanics. They will either leave the server or just boycott the mythics. We are talking about multiple guilds here! Even if Mythics are included, they won't bring many new players, but the lost of the faithfull community so far for months or years will be big.

Consider this server as a restaurant, while the food is good the regular customers will come, ocassianal bypasser will like it and will be a regular. The moment you F the food, your regulars will start coming ocassianally for few times. If they don't see a change, they will find a new place. 

Another thing T4 release shouldn't have been the place to experiment with this. Everyone was hyped from the 7 month dry period of almost nothing. People came back to the server to raid again and they were welcomed with the faceroll of 5 minutes in total raids, without even an option to wipe.

**Reply by TotallyNotMyAccount** [reply]
> If you felt my previous comment as an attack, this one might be felt as an attempt to kill you...

Yeah mate, I can feel the love and peace just oozing from you. You don't totally have a giant chip on your shoulder.

>Can you and your guildies think for long term about what is good for this server? 

more veiled attacks against my guild, stay classy

>Because already people are thinking they will F up wrath in the future with all this additional custom things in dungeons and raids. 

Meaningless fearmongering, sippery-slope fallacy.

.... aaand it goes on like this. I won't bore the rest of the people here with going through the details further. But keep on trying to confuse the issue with completely ludicrous comparisons between wow and a resturant(lol) or whatever other drivel you can come up with.

To cut through all the fluff and ad hom that you try to throw up to hide that you don't have any kind of reasonal argument:
-Additional difficulties in raids aren't blizzlike, but CC staff seems to want them because they can't balance the raids in a way that satisfies both casuals and hardcore players. Neither could blizz, so I don't fault CC staff for this one.
-If you implement additional difficulty settings and want to stay as blizzlike as you can then you have to take guidance from the extra difficulties Blizz released. This brings me back to the points in my first post, to reiterate: Heroic dungeons give better loot than normal dungeons. Wrath heroic raids give better loot than normal raids. Arena gives better loot than battlegrounds, and you get access to more of arena loot the higher your rating. Level 40 dungeons give better loot than level 20 dungeons. Boss mobs drop better loot than regular mobs. More challenge = better loot. This is at the root of this game's design and to deviate from it is to break it in catastrophic ways.

**Reply by shaszun** [reply]
You win! I'm not here to speak to a wall and repeat my self 10 times, when I even tried to explain it like to a 8 year old child. Neither of us will back up from what they believe. The only thing now is to wait what the staff decide and do.

**Reply by BoNT325** [reply]
> Because already people are thinking they will F up wrath in the future with all this additional custom things in dungeons and raids.

There's no need to worry about this. Wrath content is tuned to wrath talents and stats by default, that's why it's wrath. Vanilla and TBC being engaging/appropriately difficult with wrath tuning is the issue here. So any of this "mythic" stuff is unnecessary with the one exception of adding some sort of harder difficulty or tuning to Naxxramas and/or Eye of Eternity. Every other raid in wrath has challenge/hard mode/heroic stuff baked in. And even then, Naxx and EoE have 10 and 25 man versions for different levels of difficulty. 

So yes, worrying about custom mechanics screwing up Wrath is reaching a bit. Nothing actually needs to be custom in actual wrath content. In Burning Crusade? Absolutely. The raids need to be tuned.

**Comment by Cooties-cc** [comment]
Made a post semi replying to some posts in the discord mag/gruul section, but I dislike using disc to discuss stuff. Gets wiped off with meme posts and emojis after a day, and then just causes a cycle of complaint->reply->nowhere.

To start off, I saw some people saying that they disliked the heroic-lite custom stat gear that mythics dropped (as did majority from survey it seems), which I was also one of. I get the idea was to mimic heroic gear (same item, more stats) in later wotlk raid modes. However the gains were marginal at best, and there was not much incentive overall to push for them. You either did much harder raids for 1 extra gem, or didn't and have almost the same character value. The biggest perk was the mythic points to buy gear, which filled gaps you otherwise couldn't before next tier. Like me missing a lot of aq40 gear I wanted, but lost, passed on, or didn't drop. Reverting the stat changes I saw as a positive, since the incentive now is getting potentially 1.3x more gear per raid (going by news post about dungeons rates). There is also the high probability of a unique reward(s) for clearing all raids on this difficulty (as said about dungeons). 
_Forgot to add here that mythic raid drops **did not have the bonus stats**, only the gear from the vendor did. They were added to the item in the mail after purchase. I think many people overlooked this aspect, since all gear dropped in mythics was 'worthless' if same item was on vendor. You were already doing mythics for the points alone for stat upgrades, not the drops._ 

The tuning of raids was also brought to the attention of the server back in oct 2022. Since then there has not been a massive vocal outcry to remove these modes (only make changes to them _after_ they mentioned things like extra stats not before), and make a singular mode with the tuning we had in vanilla. At least not till _after_ t4 released this week. I can understand that people might have been expecting aq40 level of tuning, but it was clearly stated this would not be the case for normal modes. Bugs further plagued these fights (at least more so on mag) to make the fight even less technical or difficult. Things like being able to ignore clicking cubes and eat the aoe damage. Which is a valid point to raise about abilities expecting to be a wipe, and they simply aren't. However I would like to see more of these issues fixed as we go, then maybe raise tuning to the players expectations. It is the 1st week after all, and most guilds don't raid till the weekend. Maybe when more players experience the raids, more support will be for higher tunes. 

It was also stated that only well geared players would be doing mythic dungeons (by their news announcement), so it was assumed raids would be too. This makes me believe that world bosses were released prior to t4 to pad for mythics, but it ended up also making normal modes a lot easier too. Of course getting a little more stats doesn't instantly make a raid a pushover, however some world boss loot is better than t4 itself (my char will be using 4 pieces going into t5). 

Ending I would say perhaps give a little time for more guilds/players to even do said raids, let bugs get fixed, and then let mythics drop. See from there how most people are reacting to the changes, and then make suggestions more solidified from that. Asking for major changes only 2 days after a raid tier dropping seems rather rushed.

**Comment by balleny** [comment]
The big part which was missed here is simple:
In the current progression the following drop rates should apply, and could / would have been a big incentive for guilds to go for mythic, given that there would be e.g. 3 tokens and 6 items dropping:

- Maulgar: 1 token, 1 item
- Gruul: 1 token, 2 items
- Magtheridon: (vanilla) 1 token, 2 items, no bag, no head, no gem bag
- Magtheridon: (classic) 1 token, 2 items, 1 bag, 1 head, 1 gem bag

**Reply by Cooties-cc** [reply]
Yeah that struck me as odd when someone mentioned it, asking what the total loot pool per week is if you did all the raids. I had to double check the PTR, and it appeared far too high weekly totals, even for a 25 man. I would agree that increasing the pool would have been a good change for mythic versions, but I also don't fully know the limitations of the client. Something along those lines was originally described for the mythic modes, however it was presented as a higher % chance to drop than other items (staff picked 'bis' items and increased drop % chance). Everyone thought for the most part, that was a bad idea. Nothing was brought up to change or provide alt methods, at least none talked about from staff (visibly, not like 1 reply in gen chant on discord lmao). 

Currently, as you stated on the github ticket, the 3 bosses in total drop : 
9 tokens (3x chest, pants, shoulder)
6 regular drops (pool of 12/13/6)
8 badges (I actually hadn't tested again before release, but were on PTR at least [they shouldn't atm])
20 slot bag (shouldn't be in tiil 2.4)
2 heads (you missed that both ally _and_ horde can each get a turn in) [also not till patch 2.1]
Gem sack (wasn't in till 2.4, thou classic made 2 version)

That's a total per week of 17 gear items (plus fraction of item from badges), plus another 4 if you only did world bosses _once_ a week (also not including main rotations using alts). I agree that's slightly too much character income to ignore, while these raids are considered quite a bit harder than the 10 man Kara (mag being of the hardest technical t4 boss). 

[_perhaps relating to passing and transparency of info from staff to players, but not sure if off topic_] : 
I also have to laugh each time I see "community" and "guild reps" brought up as a source of server opinions. _Clearly that worked_ when it was assumed both were aware of the stat changes on mythic gear. I can also say our guild wasn't even sure who our guild rep was, and later found out it was someone that hadn't logged on in over a month+ after 70 phase was released. We had zero word of mouth from choices discussed or chosen behind closed doors. I would like to see guild reps removed as a 'source of community feedback', and just have them be the ones to connect with if there's an issue with the guild(s) directly (not server choices). Doing polls and surveys with as much as the data public is the best choice, and clearly a much more rounded answer to how things should move forwards. Not letting it be a free for all in discord's general chat. This also isn't a dig at staff, and I understand they pick what they honestly think is best for the server and players. I would just like to see a little more staff engagement with players about unique additions to the server, and how those who are doing said content might like to see done in certain ways. Not everyone can be satisfied, but can at least give a chance to be close.

**Reply by balleny** [reply]
(_I am aware that the head is faction depending, just didnt specify it_)

I like how you mention guild Rep channel. I am frequently complaining about that closed door discussion, e.g. here
https://github.com/chromiecraft/chromiecraft/discussions/4916
simply asking to make the conversations visible (read-only for everyone, and write permissions for the people currently in). I know this isnt in everyones favor but would at least allow people to reach out to their g-reps and forward feedback. Especially considering that we have more guilds forming again and potentially quite a few guilds arent even aware of these guild rep options (as it is invisible / closed door right now)

**Reply by georgeclamnis** [reply]
Previously, tickets would be closed within 10 minutes, but now they are not even taken into consideration. We understand what you are doing, but a professional team does not work this way. This is not how you manage a server and a community. Just because Honey developed the Mythics doesn't mean we should automatically say yes and want to play them. Similarly, just because a colleague develops the Mythics doesn't mean we should blindly protect them. For now, it's not a catastrophe, but it can still be corrected. We don't want to see such errors in the future. Seek the community's input before making any decision, as we are the players, the customers. I have noticed numerous comments where everyone is trying to defend their own ideas and camps. On closer observation, there are three types of camps: the CC team (with fake GitHub accounts to defend their ideas, although this is unprofessional), the most populous guild on this server trying to defend their ideas and aligning with the CC staff, and the rest of the poor community who wants to play a blizzlike experience with minimal customization and modifications, but they are ignored or downplayed by claiming they are talking nonsense (when they are right).

**Reply by IskrenPlashkov** [reply]
Frankly, the more comments I read here and on disc the less I understand the "blizzlike" side of the argument. If you truly wanted such experience you would be petitioning for the original TBC talents, professions etc. Picking and choosing when to apply certain standard is not only disingenuous but also hypocritical. In addition to that, we are discussing a feature that you clearly have no interest in. Luckily nobody is forcing you to use it. So why be so adamant to prevent others who are looking for a challenge from having it ?

**Reply by joesill** [reply]
On the "what should and shoudln't be blizzlike?" question:

We want to fight blizzlike bosses, i.e., bosses that react as and propose a challenge similar to what they offered in original TBC, and do it with wotlk talents and itemization. This is probably what attracted a large part of the audience of this server and this is what was originally made available with the tuning back in vanilla chromie. 

On the "you don't have to use this feature so don't worry about it" question:

If the development of the new features only a part of the server wants to use didn't affect the development of regular features, that would be totally ok. But given that the development of mythic raiding is an exclusive alternative to regular raid tuning, people that want the regular raid tuning will have to argue against mythics.

**Reply by IskrenPlashkov** [reply]
Fully support you on the _"We want to fight blizzlike bosses, i.e., bosses that react as and propose a challenge similar to what they offered in original TBC, and do it with wotlk talents and itemization. This is probably what attracted a large part of the audience of this server and this is what was originally made available with the tuning back in vanilla chromie."_ Which is why I advocate for increasing raid difficulty. As retail TBC veteran I know very well the initial challenge level these raids provided and believe me, what we have here certainly isnt it! Does anyone here actually enjoy the current state of Tier 4 raids ???

On your second point I also agree with one small asterisk: Server staff definitely botched the exaction **AND** timing of mythic release. That being said, when the feature is 95% done it would be foolish to scrap the whole thing 2 steps off the finish line. It is our responsibility to provide honest constructive critisism and it would be their responsibility in return to deliver a feature that the community likes and actually uses.

**Reply by balleny** [reply]
I dont know where you are looking at, but if you see the bug tracker, you see plenty of tickets regarding proper progression. there was been stuff missed, but the feedback is there.
talents on the other hand are quite a significant change, and require a lot more effort. hence, they are deemed not worth the lost dev time for content fixes.
tuning is just a matter of making this game non-trivial
